src/anemoi/training/train/train.py

```diff
@@ -308,6 +308,7 @@ def strategy(self) -> DDPGroupStrategy:
         """Training strategy."""
         return DDPGroupStrategy(
             self.config.hardware.num_gpus_per_model,
+            self.config.dataloader.read_frequency,
```

Comment:
could we think of a way to make use of the routines / groups computed by the strategy in self.datamodule?

Reply:
I managed to pass them to the dataloader directly via DDPGroupStrategy.process_dataloader() which is called by pytorch_lightning in trainer.fit(model, datamodule).